docs/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "prestart": "gatsby clean",
```

Comment:
i think i'd like to see the js formatter/linter here if at all possible, i have no opinion on which, but i'd rather enforce some standard than none

Reply:
Do you mean a formatter for the `md` files or the `package.json`?